django/db/models/base.py

```diff
@@ -210,6 +210,15 @@ def __new__(cls, name, bases, attrs):
                 # Things without _meta aren't functional models, so they're
                 # uninteresting parents.
                 continue
+ 
+        # The super class has an attribute for each sub class with the same sub class but lower cased.
+        # Check for clashes between locally declared fields and sub class name in lower cased.
+            for field in field_names:
+                if field == name.lower():
```

Comment:
I'm not so sure `name.lower()` is correct. Hopefully there is a more canonical way to get the name of that accessor.